web/src/portico/hello.ts

```diff
@@ -12,30 +12,32 @@ function get_random_item_from_array<T>(array: T[]): T {
 }
 
 const current_client_logo_class_names = new Set([
-    "client-logos__logo_akamai",
-    "client-logos__logo_tum",
-    "client-logos__logo_wikimedia",
-    "client-logos__logo_rust",
-    "client-logos__logo_dr_on_demand",
-    "client-logos__logo_maria",
+    "client-logos-div client-logos__logo_akamai",
+    "client-logos-div client-logos__logo_tum",
+    "client-logos-div client-logos__logo_wikimedia",
+    "client-logos-div client-logos__logo_rust",
+    "client-logos-div client-logos__logo_dr_on_demand",
+    "client-logos-div client-logos__logo_maria",
 ]);
 const future_client_logo_class_names = new Set([
-    "client-logos__logo_pilot",
-    "client-logos__logo_recurse",
-    "client-logos__logo_level_up",
+    "client-logos-div client-logos__logo_pilot",
+    "client-logos-div client-logos__logo_recurse",
+    "client-logos-div client-logos__logo_level_up",
 
-    "client-logos__logo_layershift",
-    "client-logos__logo_julia",
-    "client-logos__logo_ucsd",
-    "client-logos__logo_lean",
-    "client-logos__logo_asciidoc",
+    "client-logos-div client-logos__logo_layershift",
+    "client-logos-div client-logos__logo_julia",
+    "client-logos-div client-logos__logo_ucsd",
+    "client-logos-div client-logos__logo_lean",
+    "client-logos-div client-logos__logo_asciidoc",
```

Comment:
As a note: portico changes aren't relevant for the performance goal; it doesn't really matter. It's still better code, but wouldn't have been a priority in my mind.